README.md

```diff
@@ -108,7 +115,148 @@ Log in with psql shell to cnx-db
+
+Please install it using [this guide][rcloneinstall].
+
+#### Get your S3 credentials
+
+Login to your [aws console][awsconsole] with your credentials.
+
+Go to upper right corner to "My Security Credentials"
+
+![securitycredentials](https://i.imgur.com/ltnzm71.png)
+
+Now create access keys
+
+![accesskeys](https://i.imgur.com/zXe2dlA.png)
+
+Copy&Paste this access keys (key id & secret access key) into your password manager or download them. You will need them later.
+
+#### Setting up rclone
+
+We will create a new remote location named `openstax` for accessing the Openstax sandbox buckets. If not already done [install rclone][rcloneinstall] (e.g. on Mac with `brew install rclone`).
+
+* Configure rclone with and create a new remote (n):
+
+```
+rclone config
+```
+
+![rcloneconfig](https://i.imgur.com/jdUiGMP.png)
+
+* Name it e.g. `openstax` and choose Amazon S3:
+
+![rclones3](https://i.imgur.com/DGGVWAw.png)
+
+* Choose S3
+* Enter AWS credentials in next step
+* Now enter your access key id and secret access key inside:
+
+![rclonesecret](https://i.imgur.com/24yuike.png)
+
+* Choose Ohio: `us-east-2`
+* Use default entpoint
+* Choose again Ohio: `us-east-2`
+* Choose FULL_CONTROL for owner (default):
+
+![rclonefullcontrol](https://i.imgur.com/hp7zL2k.png)
+
+* Choose no server side encryption
+* No KMS ID:
+
+![rclonekms](https://i.imgur.com/pquRRqW.png)
+
+* Default storage class
+* No advanced config
+
+* At the end confirm all your settings which should look similar to this:
+
+![rcloneconfirm](https://i.imgur.com/O128Y4l.png)
+
+* quit config
+
+#### Test that rclone works
+
+Run following command to remote `openstax` look at the buckets available:
+
+```
+rclone lsd openstax:
+```
+
+You should see a list of buckets and also our test bucket `ce-rap-test`:
+
+![ce-rap-test](https://i.imgur.com/9AALpsA.png)
+
+#### How to use rclone
+
+Some examples of rclone usage.
+
+List the contents of bucket `ce-rap-test`:
+
+```
+rclone ls openstax:ce-rap-test
+```
+
+Copy all content from `ce-rap-test` to your current directory:
+
+```
+rclone copy openstax:ce-rap-test ./
+```
+
+Copy one file into the bucket:
+
+```
+rclone copy ./nothinginside.md openstax:ce-rap-test/
+```
+
+Information: Be careful with `move` and `delete` commands!
+
+#### How to mount a bucket into your local filesystem with FUSE
+
+rclone can also mount buckets into your local filesystem with FUSE.
+
+Info: On mac you need to install [FUSE for macOS][macosfuse] once.
+
+First create a local folder e.g. in your home dir where you want to mount the bucket. For example:
+
+```
+mkdir ~/s3files
+```
+
+Now mount the bucket with rclone:
+
+```
+rclone mount openstax:ce-rap-test ~/s3files
+```
+
+**Now you can use that folder as remote mount and can copy/move/delete files into s3 and out!**
+
+To stop mounting press `Ctrl-C`. Sometimes unmounting can fail. In this case use this command to unmount your mount folder manually:
```

Comment:
Ah right, you're running in the foreground and using ctrl-c to unmount.  I'm used to using `fusermount -u` so :sweat_smile: